ts/screens/main/MessagesScreen.tsx

```diff
@@ -23,15 +23,15 @@ import { MessageWithContentPO } from "../../types/MessageWithContentPO";
 type ReduxMappedProps = Readonly<{
   isLoadingMessages: boolean;
   messages: ReadonlyArray<MessageWithContentPO>;
-  services: ServicesState;
+  services: Readonly<ServicesState>;
 }>;
 
 export type OwnProps = Readonly<{
   navigation: NavigationScreenProp<NavigationState>;
 }>;
 
 interface IMessagesList {
-  item: MessageWithContentPO;
+  item: Readonly<MessageWithContentPO>;
```

Comment:
not here, but at the `IMessagesList` level (you'll need to use `type` instead of `interface`)